images/kubemci/Makefile

```diff
@@ -26,12 +26,18 @@ image-latest: image
 
 # Build and push the versioned image.
 push: image
-	gcloud beta auth configure-docker
-	docker push "$(IMG):$(VERSION)"
+	# TODO(krzyzacy): https://github.com/GoogleCloudPlatform/docker-credential-gcr/issues/21#issuecomment-365803776
+	# gcloud auth fails in CI, this image build file runs in CI only
+	# so use gcloud docker -- push for now until the above issue resolves.
+
+	# gcloud beta auth configure-docker
```

Comment:
I think we need to run `mkdir -p $HOME/.docker` first, because gcloud doesn't currently verify that directory exists when it tries to write `$HOME/.docker/config.json`. :(